### ISSUE TITLE
DateTime objects use System time zone instead of Luxon's

### ISSUE THREAD

**Issue by langalex** [cibernox/ember-power-calendar-luxon#40]
Not sure if this is intended or not:

```js
Settings.defaultZone = 'Europe/Berlin'
```

```hbs
<PowerCalendar @onSelect={{this.selectCalendarDay}} />
```

```js
@action
async selectCalendarDay(date: {
    datetime: DateTime;
  }): Promise<void> {
    console.log(
      datetime.zoneName,
    );
```

Instead of `'Europe/Berlin'`, the system time zone is logged. This can lead to issues down the line, e.g. when sending this DateTime to and endpoint including the hour and time zone. 

A workaround is to call `.setZone(Settings.defaultZone, { keepLocalTime: true })`, which changes the time zone but not the hour.

**Comment by mkszepp** [comment]
Thanks for reporting, looks like `DateTime.fromJSDate` sets always `local` as zone when we don't pass it...

To fix you issue i think we need to pass always as option `zone`.

Can you try if this returns the correct zone?
If yes, feel free to add a PR.

**Comment by langalex** [comment]
I [tried a few things](https://github.com/cibernox/ember-power-calendar-luxon/pull/41/files) but the default time zone set in the app does not reach the add-on. The  `console.log` statements always output *system time zone*. Any ideas?

**Comment by mkszepp** [comment]
hmm looks like this is a `ember-auto-import` issue... i will look to find time to check and how we can fix it..

**Comment by langalex** [comment]
interesting. if you can point me in the right direction I could also try to figure it out.

**Comment by mkszepp** [comment]
@langalex i have yesterday asked NullVoxPopuli, but it looks like we need to debug deep.
One fix should be should to move `luxon` to peerDependency (atm dependency), to reduce duplicate dependency. I have also already tested to do this, but this hasn't solved the problem.

The problem looks like we have two different instances of luxon...
Its maybe a `ember-auto-import` or a module order problem...
A fix could be to hold luxon in secret globals... but we need to look step by step.

A simple workaround could be, to make a function in meta package, which can be called from app like `DateLibary.setTimezone('UTC')` and inside the function we set the timezone. This i have tested and it works, but its not so super...
